### DISCUSSION TITLE
How to get information about empty response errors?

### DISCUSSION THREAD

**Discussion by nicekiwi** [filamentphp/filament#12119] [Help]
### Package

Panel builder

### Package Version

v3.2.61

Laravel 11
PHP 8.3

### How can we help you?

When I try to create or edit Resource relations, I get a "EMPTY RESPONSE" error in dev tools, but no details, nothing in laravel.log, no console.log details, how can I find out whats wrong?

After clicking "create" or "create and create another" the loading icon will spin for a few seconds, (this is running locally so it should be instant), then stop with the EMPTY RESPONSE in the console.

```
/* ItemSponsorsRelationManager */

protected static string $relationship = 'itemSponsors';

    public function isReadOnly(): bool
    {
        return false;
    }

    public function table(Table $table): Table
    {
        return $table
            ->recordTitleAttribute('sponsor.name')
            ->columns([
                Tables\Columns\TextColumn::make('sponsor.name'),
                Tables\Columns\TextColumn::make('note'),
            ])
            ->filters([
                //
            ])
            ->headerActions([
                Tables\Actions\CreateAction::make()
                    ->form(fn (Tables\Actions\CreateAction $action): array => [
                        Forms\Components\TextInput::make('item_id')
                            ->required()
                            ->hidden()
                            ->default($this->getOwnerRecord()->id),
                        Forms\Components\Select::make('sponsor_id')
                            ->label('Sponsor')
                            ->options(fn (callable $get) => Sponsor::all()->pluck('name', 'id')->toArray())
                            ->required(),
                        Forms\Components\Textarea::make('note')
                            ->label('Note'),
                    ]),
            ])
            ->actions([
                Tables\Actions\DeleteAction::make(),
            ])
            ->bulkActions([
                Tables\Actions\BulkActionGroup::make([
                    Tables\Actions\DeleteBulkAction::make(),
                ]),
            ]);
    }
```

![CleanShot 2024-03-31 at 21 31 41@2x](https://github.com/filamentphp/filament/assets/738975/c5043dfd-a8a6-456a-9c17-cbc83264b1f1)

**Comment by leandrocfe** [comment]
What about the other resources?
Use should mutate the data instead of using a hidden input
https://filamentphp.com/docs/3.x/actions/prebuilt-actions/create#customizing-data-before-saving

**Reply by nicekiwi** [reply]
@leandrocfe which other resources? 

I have another form with the same issue:

```
return $form
            ->schema([
                Forms\Components\TextInput::make('tmdb_id')
                    ->required()
                    ->maxLength(255),
    ]);
```

No hidden fields, just empty response and no way to diagnose what's wrong.

**Reply by leandrocfe** [reply]
Are Events/ Films/ Genres resources working without issues?

**Reply by nicekiwi** [reply]
Some do, it seems to be intermittent depending on the individual item. But again zero feedback given by filament as to what was wrong. e.g. one specific film resource cannot be saved even when it's just the ID in the form, but other existing resources can be edited and saved and created without issue. 

There is no feedback, and its incredibly irritating.

**Reply by nicekiwi** [reply]
For example: https://share.cleanshot.com/9QJRPXF1

**Reply by nicekiwi** [reply]
and the same again with only the ID in the form. https://share.cleanshot.com/GLR3Rn3r

**Reply by leandrocfe** [reply]
Weird.. Could you create a fresh Laravel/Filament install and check if you have the same issue please?
